docs/CLI.md

```diff
@@ -100,7 +100,7 @@ jest --update-snapshot --detectOpenHandles
 
 ## Options
 
-_Note: CLI options take precedence over values from the [Configuration](Configuration.md)._
+:::note CLI options take precedence over values from the [Configuration](Configuration.md). :::
```

Comment:
@SimenB  prettier seems to be making it one line and this does not work , as admonition work only when we place the it like this 
```
:::note
text
:::
```

Reply:
https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

Reply:
> https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

thanks a lot , 😀